.github/workflows/contributors.yml

```
@@ -0,0 +1,18 @@
+on:
+    push:
+        branches:
+            - main
+            - "*"
+
+jobs:
+    contrib-readme-job:
+        runs-on: ubuntu-latest
+        name: A job to automate contrib in readme
+        permissions:
+          contents: write
```

yamllint

[warning] wrong indentation: expected 12 but found 10

(indentation)